api/v3/Case.php

```diff
@@ -416,6 +416,9 @@ function _civicrm_api3_case_read($caseId, $options) {
   if ($dao->find(TRUE)) {
     $case = array();
     _civicrm_api3_object_to_array($dao, $case);
+
+    _civicrm_api3_custom_data_get($case, 'Case', $caseId);
```

Comment:
Then I don't know why that was required.... The syntax conformance test already tests get & create for all entities defined in the static map (I thought).